test/test_proxy_tensor.py

```diff
@@ -0,0 +1,173 @@
+# Owner(s): ["oncall: fx"]
```

Comment:
Might be good to have these and the tests from `test_fx_experimental` in one file instead of two